engine/src/flutter/lib/web_ui/test/canvaskit/image_golden_test.dart

```diff
@@ -96,7 +97,7 @@ class BitmapTestCodec extends TestFileCodec {
   Future<ui.Codec> createCodecFromTestFile(String testFile) async {
     final DomHTMLImageElement imageElement = createDomHTMLImageElement();
     imageElement.src = testFile;
-    setJsProperty<String>(imageElement, 'decoding', 'async');
+    imageElement['decoding'] = 'async'.toJS;
```

Comment:
Consider adding this as a property on `DomHTMLImageElement`.

Reply:
ack - it was already there!!!